components/camel-snmp/src/main/java/org/apache/camel/component/snmp/SnmpProducer.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * 
+ */
```

Comment:
I have copy the intestation from another file but i have the same error.

What do i do wrong?